lib/backend/etcd/etcd.go

```diff
@@ -117,6 +117,7 @@ func (c *EtcdClient) Update(d *model.KVPair) (*model.KVPair, error) {
 	// If the request includes a revision, set it as the etcd previous index.
 	options := etcd.SetOptions{PrevExist: etcd.PrevExist}
 	if d.Revision != nil {
+		log.Infof("Updating etcd revision from: %v to: %v", options.PrevIndex, d.Revision.(uint64))
```

Comment:
curious why do we want to remove this one but keep the one in `compat`? Only some resoures/objects go through the extra layer of `compat`, so I think it would look inconsistent if we only log the revision update for the ones that go through `compat` (Block, Node and Profile as of now)